### ISSUE TITLE
Integrate on my local project

### ISSUE THREAD

**Issue by jenistar90** [traitify/traitify-js-widgets#3]
Basically, I understand about the Traitify.

I include the Traitify's Library like this.
<script src="https://cdn.traitify.com/js/widgets/v1.js"></script>

But to translate the traitify, I have to use Traitify JS Widget and put it on my local.
https://github.com/traitify/traitify-js-widgets

But on that Widget, I don't see widgets/v1.js file.

Then what JS library I have to include on this Widget if I put this on local?

**Comment by carsonwright** [comment]
If you would like v1 without compiling it go to the cdn and download it:
http://cdn.traitify.com/js/widgets/v1.js

I would let the cdn load it though and then use the js attached in the patches folder. Remember it is purely an example of how to use translations and will have to be filled out with the correct data. If you have more questions please let me know

**Comment by jenistar90** [comment]
Thanks for your replying, Carson
I just download v1.js and tried on my local. And create the 'patches' folder and put translation.js file in there. Of course, I translate all the words.
But I failed.
So I open v1.js and search the word 'translation.js'. But I couldn't find anything. Maybe, current CDN v1.js doesn't seems complied as new one what including the translation.
Any idea? This is my current folder structure. http://screencast.com/t/9zlVXtLhle

Everything is working well except translation.
This is the screenshot of translation.js file : http://screencast.com/t/7yNwoEbt

One thing more... I answer on Traitify's slider, but I didn't complete the answer. Maybe I answer half and close the browser. And if I open again, it doesn't keep the slider what I closed before.
Other issue is... I finish to answer the questions and see the result of Personality. Then I close the browser and open it again. Sometimes, it askes to answer the all questions again, just not keeping the result what I answered. Is it problem of since I am using sand-box?

Regard.
